### ISSUE TITLE
Artifactory

### ISSUE THREAD

**Issue by cohiggins** [warrenchristian1telus/moodle#12]
- Get images on to the Artifactory repo; build pipelines
- Migrate to BCGOV instance 
- Investigate early access to additional features

**Comment by cohiggins** [comment]
FuXing to confirm Warren/Kunal

**Comment by cohiggins** [comment]
Warren, Allan, Kousthubha (maybe Kunal) to meet with JFROG team on April 9

**Comment by cohiggins** [comment]
Waiting on update from DevExchange

**Comment by warrenchristian1telus** [comment]
Moodle test namespace has been configured to use TELUS Artifactory as the image pull source. I'm investigating options to migrate this to the DevOps Artifactory instance,

**Comment by cohiggins** [comment]
Closing; will create new issue to handle additional work as part of setting up the cicd pipeline.
